doc/api-extensions.md

```diff
@@ -309,3 +309,7 @@ a specific storage pool.
 This adds support for editing a container metadata.yaml and related templates
 via API, by accessing urls under /1.0/containers/NAME/metadata. It can be used
 to edit a container before publishing an image from it.
+
+## container\_snapshot\_stateless\_migration
```

Comment:
So the previous behavior was to always transfer the state?

I somehow thought (based on the bug mostly) that we were always doing stateless right now and that this effectively adds the needed code to support state transfer.

If that's the case, then the name should be container_snapshot_stateful_migration I guess.